docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
```

Comment:
Add a list of the different existing LDAP schema's we're going to handle.  Provide a brief description and links.